storage_broker/src/bin/storage_broker.rs

```diff
@@ -96,6 +96,7 @@ enum Message {
 
 impl Message {
     /// Convert proto message to internal message.
+    #[allow(clippy::result_large_err, reason = "TODO")]
```

Comment:
do we need these `reason =` fields? to me it seems they can be removed.

Reply:
I added them to make it clear that I have not yet considered whether this allow is good or not